perftracker/models/regression.py

```diff
@@ -38,10 +39,10 @@ def save(self):
         super(RegressionModel, self).save()
 
     def ptGetJobs(self):
```

Comment:
As I understand now you should have 2 methods:

ptGetAllJobs() - list all the jobs with given regression tag
ptGetLinkedJobs() - list the jobs included into regression (there can be jobs with given tag but manually excluded)